packages/server/src/index.ts

```diff
@@ -1,6 +1,6 @@
 export * from './assertNotBrowser';
 export * from './http';
-export * from './router';
+export * from './router.legacy';
```

Comment:
I've used the `./deprecated`-folder for this but I'm with above too

```suggestion
export * from './router';
export * from './deprecated/LegacyRouter';
```